### DISCUSSION TITLE
Model Space as Current View for New File

### DISCUSSION THREAD

**Discussion by thaiguer** [DomCR/ACadSharp#492] [Q&A] [answered]
Hello,
When creating new files, the default view when opened by CAD software is Layout1. Is there a way to change it to Model?

### Current behavior when first opened by CAD software:
![image](https://github.com/user-attachments/assets/4186ea70-a909-481b-ba40-15f3bfb4ea36)

### Desired default view when first opened by CAD software:
![image](https://github.com/user-attachments/assets/1add2656-c3f3-44a1-aa5b-2df2861338b2)

### Grid:
Also, is there an option to turn off the grid? Right now, the grid is always on.

**Comment by DomCR** [comment, marked as answer]
Hi @thaiguer,

You have an option in the header, `CadHeader.ShowModelSpace`.

For the grid, you have to modify the `VPort.ShowGrid`.

Hope this helps!

**Reply by thaiguer** [reply]
To Show Model Space, this works perfectly:
```
CadDocument newDoc = new CadDocument();
newDoc.Header.ShowModelSpace = true;
```

But for the grid, can you show a code example?

**Reply by DomCR** [reply]
Here you go:

`this.Document.VPorts[VPort.DefaultName].ShowGrid = false;`
